variables.tf

```diff
@@ -111,4 +177,45 @@ locals {
     for mysql_flexible_server_firewall_rule in keys(var.mysql_flexible_server_firewall_rule) :
     mysql_flexible_server_firewall_rule => merge(local.default.mysql_flexible_server_firewall_rule, var.mysql_flexible_server_firewall_rule[mysql_flexible_server_firewall_rule])
   }
+
+  /**
+    compare and merge custom and default values
+  */
+  postgresql_flexible_server_values = {
```

Comment:
don't duplicate the compare block again so please move this to line 147.

Reply:
ok, done